.github/workflows/check_black.yml

```diff
@@ -0,0 +1,15 @@
+
+name: Check Black formatting
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: psf/black@stable
+        with:
+          options: "-l 79 --check"
+          src: "."
```

Comment:
@jdebacker. Will you put a blank line at the end of this file? This is a small thing, but GitHub flags it.